functionary/ui/tables/schedule_task.py

```diff
@@ -0,0 +1,44 @@
+import django_tables2 as tables
+from django.urls import reverse
+from django.utils.html import format_html
+
+from core.models.scheduled_task import ScheduledTask
+
+
+class ScheduledTaskTable(tables.Table):
```

Comment:
I'm noting this here only because the template is gone...

We lost the Create buttons on the Schedules and Workflows pages.  Those will have to have their own template and not use the default so that you can add the button back.